repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,21 @@
+#########################
+# Gatling Configuration #
+#########################
+
+gatling {
+  charting {
+    indicators {
+      percentile1 = 50      # Value for the 1st percentile to track in the reports, the console summary and Graphite
+      percentile2 = 95      # Value for the 2nd percentile to track in the reports, the console summary and Graphite
+      percentile3 = 99      # Value for the 3rd percentile to track in the reports, the console summary and Graphite
+      percentile4 = 99.9    # Value for the 4th percentile to track in the reports, the console summary and Graphite
+    }
+  }
+  data {
+    writers = [console, file, graphite] # The list of DataWriters to which Gatling write simulation data (currently supported : console, file, graphite, jdbc)
```

Comment:
Gatling shouldn't care. iptables on the InfluxDB host is setup to drop packets by default, so the Gatling writer should be able to establish a TCP connection, but all of the packets it sends through that connection will be dropped. The Graphite line protocol is an exceptionally simple application-level protocol -- it doesn't respond to the data it receives in any way.